tests/create-an-actual-host.cc

```diff
@@ -27,10 +29,17 @@ struct test_host : clap::helpers::Host<clap::helpers::MisbehaviourHandler::Termi
    void requestCallback() noexcept override {};
 };
 
+using test_plugin_proxy = clap::helpers::PluginProxy<clap::helpers::MisbehaviourHandler::Terminate,
+                                                     clap::helpers::CheckingLevel::Maximal>;
+
 CATCH_TEST_CASE("Create an Actual Host")
 {
    CATCH_SECTION("Test Host is Creatable")
    {
       CATCH_REQUIRE(std::is_constructible<test_host>::value);
    }
+   CATCH_SECTION("Test Plugin Proxy is Creatable")
```

Comment:
Minor but: Should this be in a separate TEST_CASE since its not "Create an Actual host" related?